feature_engine/creation/__init__.py

```diff
@@ -2,6 +2,8 @@
 The module creation includes classes to create new variables by combination of existing
 variables in the dataframe.
 """
+
+from .custom_functions import CustomFunctions
```

Comment:
this PR is a bit of a mix. We have the changes of mathfeatures, which were not yet merged, and then changes to other files. 

Ideally, we'd only have updates to current files here, and all changes to math features in the mathfeatures pr. Otherwise, we can't merge here until we approve the other PR.

Reply:
file removed from this branch